docs/reference/api/platform/Callback.md

```diff
@@ -27,3 +27,156 @@ Here is an example that uses everything discussed in the [introduction to callba
 
 - [Platform overview](/docs/development/reference/platform.html).
 - [Introduction to callbacks](/docs/development/reference/platform.html#callbacks).
+
+#### Create callbacks
+
+First, you need to understand the syntax of the Callback type. The Callback type is a templated type parameterized by a C++ function declaration:
+
+``` c++
+// Callback</*return type*/(/*parameters*/)> cb;
+Callback<int(float)> cb;          // A callback that takes in a float and returns an int
+Callback<void(float)> cb;         // A callback that takes in a float and returns nothing
+Callback<int()> cb;               // A callback that takes in nothing and returns an int
+Callback<void(float, float)> cb;  // A callback that takes in two floats and returns nothing
+```
+
+You can create a Callback directly from a C function or function pointer with the same type:
+
+``` c++
+void dosomething(int) {
```

Comment:
Great to move the content around but this still violates the no foo/bar rule imo. Make sure to keep another task to make the example useful or tangible

Reply:
I think I'd rather just do that now.